data/json/items/gun/22.json

```diff
@@ -444,5 +444,56 @@
     ],
     "price_postapoc": "10 USD",
     "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ]
+  },
+  {
+    "id": "rossi_gallery",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "pump action varmint rifle" },
+    "description": "A lightweight, wood framed pump action rifle, holding 15 rounds of .22 with an 18 inch barrel.  While pump actions tend to be largely shotguns, they can be made in calibers such as .22 LR.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rossi_gallery",
+        "name": { "str": "Rossi Gallery", "str_pl": "Rossi Galleries" },
+        "description": "A modern take on the Winchester 1890, the Rossi Gallery is a .22 LR rifle, notably using a pump action.  Much like a pump action shotgun, the magazine tube is below it's 18 inch barrel and can hold 15 rounds.  While pump actions are quite often associated with shotguns, the mechanism can be applied to rifles as well, though in practice it's limited to .22 LR."
+      }
+    ],
+    "weight": "2344 g",
+    "volume": "2054 ml",
+    "longest_side": "913 mm",
+    "barrel_length": "457 mm",
+    "price": "394 USD",
+    "price_postapoc": "10 USD",
+    "to_hit": -1,
+    "material": [ "steel", "wood", "aluminum" ],
+    "symbol": "(",
+    "color": "brown",
+    "range": 7,
+    "ammo": [ "22" ],
+    "skill": "rifle",
+    "ranged_damage": { "damage_type": "bullet", "amount": 3 },
+    "dispersion": 90,
```

Comment:
This seems a little too accurate for this type of firearm; 90 is the same as in game sniper’s weapons. I’ve seen reviews that mention off-hand accuracy as

> two inches at 25 yards.

I don’t know what that converts over to in MOA, so it could be 0.9, but it sounds unlikely. Please correct me if I am wrong about this matter.

Reply:
This was just using the value on the 39A, I'll fix it